.gitignore

```diff
@@ -31,3 +31,263 @@ database.trace.db
 
 # Ignore Mac OS X
 **/.DS_Store
+.idea/$PRODUCT_WORKSPACE_FILE$
```

Comment:
Is there anything in `./idea` we need? You can change all of this to be `./idea` and it will ignore the entire directory